### PR TITLE
[docs]: Fix documentation for the Bounded Box Component

### PR DESCRIPTION
**Pull request by bhojrajnarwae**
I have updated the the changes that were mentioned in issue #11057. The following changes are included:

1. The change in format updates
2. The change in content updates

The changes were implemented in two files :-
Next - ToolJet/docs/docs/widgets/bounded-box.md
Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/bounded-box.md

### CODE REVIEW

File: docs/docs/widgets/bounded-box.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ id: bounded-box
 title: Bounded Box
 ---
 
+Here's the updated version with the requested changes:
```
Comment (RudraPratik30) [inline docs/docs/widgets/bounded-box.md:6]: Hey @bhojrajnarwae, you don't need to add this to the documentation. Please remove it.

File: docs/docs/widgets/bounded-box.md [hunk 2]
```diff
@@ -131,25 +135,36 @@ A Tooltip is often used to specify the extra information when the user hovers th
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
+
+Here is the updated version of the "Devices" section with the requested changes:
```
Comment (RudraPratik30) [inline docs/docs/widgets/bounded-box.md:140]: Same here, you don't need to add this to the documentation. Please remove it.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @bhojrajnarwae please make the mentioned changes in both versions

File: docs/docs/widgets/bounded-box.md (hunk 2 above)
Comment (bhojrajnarwae) [reply docs/docs/widgets/bounded-box.md:131]: @RudraPratik30 I have made the required changes as per given instructions. please check it if everything is fine.

File: docs/docs/widgets/bounded-box.md (hunk 1 above)
Comment (bhojrajnarwae) [reply docs/docs/widgets/bounded-box.md]: @RudraPratik30 I have made the required changes as per given instructions. please check it if everything is fine.

File: docs/docs/widgets/bounded-box.md (hunk 2 above)
Comment (bhojrajnarwae) [reply docs/docs/widgets/bounded-box.md]: @RudraPratik30 please check this PR and and let me know if any change to be made or everything is good to go for merging my PR.

**Comment by bhojrajnarwae** [comment]
@RudraPratik30 please check this PR and and let me know if any change to be made or everything is good to go for merging my PR.

File: docs/docs/widgets/bounded-box.md [hunk 3]
```diff
@@ -29,10 +29,10 @@ A **Bounded Box** is an infinitely customizable image annotation component that
 | :----------- | :----------- | :----------------- |
 | Image URL | The URL or image data to show it on the component. | Get the image URL dynamically from database: `{{queries.queryname.data[0].url}}` or use [image's base64 data](/docs/how-to/loading-image-pdf-from-db/) |
 | Default value | The data that will load the default bounded boxes over the image when the app is loaded. | Array of objects. Check the [Default value](#default-value) data properties |
-| Selector | The bounded box support selection using rectangle or point. | Click **Fx** to set the value `RECTANGLE` or `POINT` |
+| Selector | The bounded box support selection using rectangle or point. | Click **fx** to set the value `RECTANGLE` or `POINT` |
 | List of labels | The list of label that will be displayed in the dropdown while selection in the bounded-box. | Labels in array format: `{{['Tree', 'Car', 'Stree light']}}` |
 
-#### Default value
+### Default value
```
Comment (RudraPratik30) [inline docs/docs/widgets/bounded-box.md:35]: hey this must be a h4 only, can you please revert the changes here?

File: docs/docs/widgets/bounded-box.md [hunk 4]
```diff
@@ -131,25 +131,33 @@ A Tooltip is often used to specify the extra information when the user hovers th
...
-## Layout
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | 
-| :----------- | :----------- | 
-| Desktop | Toggle to show or hide the component in the desktop view. Dynamically configure the value by clicking on `Fx` and entering a logical expression that results in either true or false. Alternatively, the  values can be set to **`{{true}}`** or **`{{false}}`**. |  
-| Mobile | Toggle to show or hide the component in the desktop view. Dynamically configure the value by clicking on `Fx` and entering a logical expression that results in either true or false. Alternatively, the  values can be set to **`{{true}}`** or **`{{false}}`**. | 
 
 </div>
+<br/>
+<hr/>
```
Comment (RudraPratik30) [inline docs/docs/widgets/bounded-box.md:148]: Can you pls use `----` for adding the divider instead of hr tag?

File: docs/docs/widgets/bounded-box.md [hunk 5]
```diff
@@ -131,25 +131,33 @@ A Tooltip is often used to specify the extra information when the user hovers th
...
+<div style={{paddingTop:'24px'}}>
 
 ## Styles
 
 | <div style={{ width:"100px"}}> Style  </div>  | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"100px"}}> Expected Value </div> |
 | :----------- | :----------- | :----------- |
-| Visibility | Toggle on or off to control the visibility of the component when the app is loaded. |  **`{{true}}`** or **`{{false}}`**, By default, it's set to `{{true}}` |
-| Disable | Toggle on to disable the component. | **`{{true}}`** or **`{{false}}`**, By default, it's set to `{{false}}` |
-| Box shadow | Sets the add shadow effects around a component's frame. You can specify the horizontal and vertical offsets(through X and Y sliders), blur and spread radius, and color of the shadow. | Values that represent x,y, blur, spread and color. Ex: `9px 11px 5px 5px #00000040` |
+| Visibility | Toggle on or off to control the visibility of the component when the app is loaded. |  **`{{true}}`** or **`{{false}}`**, By default, it's set to **`{{true}}`** |
```
Comment (RudraPratik30) [inline docs/docs/widgets/bounded-box.md:156]: Can you please add full stops here and wherever necessary in the doc?

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @bhojrajnarwae can you please make the mentioned changes in both the versions?

**Comment by bhojrajnarwae** [comment]
@RudraPratik30 I have made the mentioned changes as follows:
1. Reverted the the Default value to h4.
2. Replace the hr tag with ---- for adding the divider.
3. Also added full stops  wherever necessary in the doc.

The following changes are done in following files:
Next - ToolJet/docs/docs/widgets/bounded-box.md
Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/bounded-box.md

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me!
